applications/rq_access_eval/Cargo.toml

```diff
@@ -0,0 +1,14 @@
+[package]
+name = "rq_access_eval"
+version = "0.1.0"
+authors = ["Klim Tsoutsman <klim@tsoutsman.com>"]
+description = "Run queue access benchmark"
+edition = "2021"
+
+[dependencies]
+app_io = { path = "../../kernel/app_io" }
+apic = { path = "../../kernel/apic" }
```

Comment:
this has now been replaced with the `cpu` crate, so we should use that instead to be arch-agnostic